### ISSUE TITLE
add expected score for `run_knn_score_model`?

### ISSUE THREAD

**Issue by repst** [cgpotts/cs224u#104]
Hi @cgpotts,

I've just done all of the assignments (except the original system) in `hw_wordrelatedness.ipynb` and related notebooks and have a couple of thoughts that may be useful to share.

- First, it is great that you have open-sourced the class material. Thank you!
- I noticed that in the **Learned distance functions** section, the function `run_knn_score_model` that students are asked to write is not tested at all. It could be helpful to check the output score so that students know they have written it correctly. You could make `train_test_split` deterministic by setting `shuffle=False`. Another option would be to add a note about what approximate score one should expect.
- In `vsm_01_distributional.ipynb` your `proper_cosine` function looks to be returning the [angular distance](https://en.wikipedia.org/wiki/Cosine_similarity). Calling it `proper_cosine` may be confusing to some unless that's a standard name for it.

Looking forward to going through the next notebooks!

**Comment by cgpotts** [comment]
Thanks for this! This assignment will be completely redone for the Spring 2023 version of the course. The above insights are great, but this whole question is now factored out (though knn will remain an option for system development).
